vllm_ascend/patch/patch_commnicator.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2025 Huawei Technologies Co., Ltd. All Rights Reserved.
+# This file is a part of the vllm-ascend project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This file is used to monkey patch communicator in vllm to support ascend.
+# Remove this file when vllm support by
+# https://github.com/vllm-project/vllm/pull/11324.
+
+from vllm.distributed.parallel_state import GroupCoordinator
```

Comment:
unrelated but just curious: should vllm be a dependency of vllm-ascend as oneline in requriement and pyproject?

Reply:
emm. Let's have a try.

While IMO, it may raises error because there is no CPU version of pytorch on pypi.

Reply:
no warries, we can do it in followup